src/Application.vue

```diff
@@ -379,6 +379,11 @@ export default {
           return
         }
 
+        Fliplet.Analytics.trackEvent({
```

Comment:
Have you confirmed this works? I'm not entirely sure if `Fliplet.Analytics` works in widget interface. I think it's only for apps. You'll need to send event tracking to the GA account for Fliplet Studio.